acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/clique/CliqueDiscardRpcAcceptanceTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.clique;
+
+import tech.pegasys.pantheon.tests.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.pantheon.tests.acceptance.dsl.node.PantheonNode;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class CliqueDiscardRpcAcceptanceTest extends AcceptanceTestBase {
+
+  @Test
+  public void shouldDiscardVotes() throws IOException {
+    final String[] initialValidators = {"miner1", "miner2"};
+    final PantheonNode minerNode1 =
```

Comment:
Think this would be nice, but the file creation need to done after nodes are started once we know all the node addresses. So this list of validators are defining the config using node names to be created later when we have the details.
So perhaps we pass in a GenesisFile creator strategy or config instead.